### PR TITLE
Add launchpad transactions to profile

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9420

## Description of Changes
- Revamped "My community stakes" page to now be a "transactions history" page with only the transaction history of user
- Moved "My Stakes" tab of the older "My community stakes" page to a "My Tokens" tab on the public profile view
- Added launchpad trade transactions to both the new "My Tokens" and "Transactions history" sections
- Reorganized data display for each transaction in the new "My Tokens" and "Transactions history" sections to be more compact

## "How We Fixed It"
N/A

## Test Plan
1. Ensure references to `/myCommunityStakes` are removed. 
2. Ensure you can visit `/myTransactionsHistory` page from profile dropdown and via URL and ensure it works on logged in state and returns "Page not found" in logged out state
3. Ensure you can see both the launchpad and stake transaction categories in the "/myTransactionsHistory" page
4. Ensure you can see "My Tokens" tab in yours and the public profile view of any user
5. Ensure you see both stake/launchpad assets in the "My Tokens" tab if user has bought both asset types.

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: libs/model/src/community/GetTransactions.query.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import type { Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { QueryTypes } from 'sequelize';
+import { models } from '../database';
+
+export function GetTransactions(): Query<typeof schemas.GetTransactions> {
+  return {
+    ...schemas.GetTransactions,
+    auth: [],
+    secure: false,
```
Comment (Rotorsoft) [inline libs/model/src/community/GetTransactions.query.ts:10]: Should we secure this? At least to check if the user is authenticated?

**Review by Rotorsoft — APPROVED** [review]
The query looks fine, just one comment

File: libs/model/src/community/GetTransactions.query.ts (hunk 1 above)
Comment (mzparacha) [reply libs/model/src/community/GetTransactions.query.ts:10]: It was required to be viewable in the public user profile so I left it like this.

File: packages/commonwealth/client/scripts/views/components/Profile/ProfileActivity/ProfileActivityRow/ProfileActivityRow.tsx [hunk 2]
```diff
@@ -158,31 +158,36 @@ const ProfileActivityRow = ({ activity }: ProfileActivityRowProps) => {
...
-                  }
-                  await navigator.clipboard.writeText(
-                    `${domain}/${communityId}/discussion/${comment.thread?.id}?comment=${comment.id}`,
-                  );
+                onClick: () => {
+                  const handleAsync = async () => {
+                    if (isThread) {
+                      await navigator.clipboard
+                        .writeText(`${domain}/${communityId}/discussion/${id}`)
+                        .catch(console.error);
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/Profile/ProfileActivity/ProfileActivityRow/ProfileActivityRow.tsx:166]: we have saveToClipboard

**Review by masvelio — APPROVED** [review]
1. I have "all addresses" selected but it shows that I dont have transactions in any community. But I do have. On the other hand, when I select address where I dont have transaction it also says that I dont have transactions in ANY community. I think we should have two different messages.
- this address does not have any transactions
- you dont have transactions in any community 

2. Also, search should work within selected address or within all my addresses? That's not clear for me as a user.

https://github.com/user-attachments/assets/16703351-e8f7-422e-be97-7ef8246c4c25

3. Mobile is very tight 

image

**Comment by mzparacha** [comment]
Thanks for the review @masvelio. Actually, I just moved some code around so the bugs/CQ are from there. I will attempt to fix the bugs and will create a followup for the CQ issues (as I don't have bandwidth to handle those ATM).

File: packages/commonwealth/client/scripts/views/components/Profile/ProfileActivity/ProfileActivityRow/ProfileActivityRow.tsx (hunk 2 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/components/Profile/ProfileActivity/ProfileActivityRow/ProfileActivityRow.tsx]: merged master just now and this seems to have been replaced with a newer component in another PR.

**Pull request merged**